docs/databases/uploads.md

```diff
@@ -47,7 +47,7 @@ When people upload a CSV to a collection, Metabase will:
 
 When you upload data, Metabase will try to detect whether the uploaded CSV file includes a column with an incrementing ID. The check is pretty basic: it'll check for a column named `id` or `pk` that has integers for each row in the uploaded CSV. Each integer in this column should be unique, so if your CSV includes duplicate ID integers, the column won't work as [an entity key](../data-modeling/field-types.md#list-of-metabase-field-types).
 
-If Metabase doesn't detect a primary key, it will create an ID column for you, including this ID column as the first (left-most) column of the uploaded CSV table.
+If Metabase doesn't detect a primary key, it will create an ID column for you, including this ID column as the first (left-most) column of the uploaded CSV table: `_mb_row_id`.
```

Comment:
Since [this PR](https://github.com/metabase/metabase/pull/36249) we now unconditionally create the column, and it is always the primary key.
```suggestion
Metabase will create a primary key column called `_mb_row_id` as the first (left-most) column of the uploaded CSV table.
```

Reply:
I'm not sure there's a clearer way to say "automatically generated integer primary key column". But an example is  in postgres, the column is created with `bigserial primary key` as its constraint.